test/test_hybridsim.cpp

```diff
@@ -3,17 +3,17 @@
 #include <gtest/gtest.h>
 #include <memory>
 
-using namespace dd::literals;
+using namespace qc::literals;
 
 TEST(HybridSimTest, TrivialParallelDD) {
     auto quantumComputation = [] {
-        auto quantumComputation = std::make_unique<qc::QuantumComputation>(4);
-        quantumComputation->emplace_back<qc::StandardOperation>(4, 2, qc::H);
-        quantumComputation->emplace_back<qc::StandardOperation>(4, 1, qc::H);
-        quantumComputation->emplace_back<qc::StandardOperation>(4, dd::Controls{2_pc, 1_pc}, 0, qc::X);
-        quantumComputation->emplace_back<qc::StandardOperation>(4, 1, qc::I); // some dummy operations
-        quantumComputation->emplace_back<qc::StandardOperation>(4, 1, qc::I);
-        return quantumComputation;
+        auto qc = std::make_unique<qc::QuantumComputation>(4);
+        qc->emplace_back<qc::StandardOperation>(4, 2, qc::H);
+        qc->emplace_back<qc::StandardOperation>(4, 1, qc::H);
+        qc->emplace_back<qc::StandardOperation>(4, qc::Controls{2_pc, 1_pc}, 0, qc::X);
+        qc->emplace_back<qc::StandardOperation>(4, 1, qc::I); // some dummy operations
+        qc->emplace_back<qc::StandardOperation>(4, 1, qc::I);
```

Comment:
consider replacing these emplace_back calls with the newly introduced convenience functions